sdk/include/aditof/camera_definitions.h

```diff
@@ -56,9 +56,16 @@ struct CameraDetails {
     ConnectionType connection;
 
     /**
-     * @brief The range of the camera in mm for the operating mode
+     * @brief The maximum extremity of the range of the camera in mm for the
```

Comment:
The maximum depth the camera can measure in the current operating mode. Distance is in millimeters.

Reply:
done, thanks